src/javascript/binary/base/header.js

```diff
@@ -205,6 +206,7 @@ const Header = (() => {
 
             const validations = {
                 authenticate   : () => +get_account_status.prompt_client_to_authenticate,
+                under_review   : () => get_account_status.status.indexOf('under_review') !== -1,
```

Comment:
the functions here are ordered alphabetically, could you move it after tnc, before unwelcome?

Reply:
Ok. 👍